setup.py

```diff
@@ -4,18 +4,20 @@
 from setuptools import setup, find_packages
 
 setup(
-    name='canonicalwebteam.http',
-    version='0.1.6',
-    author='Canonical webteam',
-    author_email='webteam@canonical.com',
-    url='https://github.com/canonicalwebteam/http',
+    name="canonicalwebteam.http",
+    version="0.1.6",
+    author="Canonical webteam",
+    author_email="webteam@canonical.com",
+    url="https://github.com/canonicalwebteam/http",
```

Comment:
not from you but link is wrong https://github.com/canonical-webteam/http (- missing)